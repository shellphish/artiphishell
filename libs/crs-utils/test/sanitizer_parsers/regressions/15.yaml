focus_repo_path: "/src/sqlite3/"
language: c
sanitizer: address
exit_code: 77
expected_organizer_significance: 212 # RecognizedNonSanitizerNotableCrash
expected_organizer_crash_description: "LibFuzzer Generic crash and/or Jazzer Sanitizer Crash"
expected_parser: 'asan'
expected_sanitizer: libFuzzer
expected_crash_type: 'deadly signal'