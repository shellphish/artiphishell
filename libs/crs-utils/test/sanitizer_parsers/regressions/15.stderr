+ '[' '!' -v TESTCASE ']'
+ '[' '!' -f /work/pov_input ']'
+ export RUN_FUZZER_MODE=interactive
+ RUN_FUZZER_MODE=interactive
+ export FUZZING_ENGINE=libfuzzer
+ FUZZING_ENGINE=libfuzzer
+ export SKIP_SEED_CORPUS=1
+ SKIP_SEED_CORPUS=1
+ run_fuzzer customfuzz3 /work/pov_input
vm.mmap_rnd_bits = 28
/out/customfuzz3 -rss_limit_mb=2560 -timeout=25 /work/pov_input -timeout_exitcode=0 < /dev/null
INFO: Running with entropic power schedule (0xFF, 100).
INFO: Seed: 1423949940
INFO: Loaded 1 modules   (74977 inline 8-bit counters): 74977 [0x555555cd4110, 0x555555ce65f1), 
INFO: Loaded 1 PC tables (74977 PCs): 74977 [0x555555ce65f8,0x555555e0b408), 
/out/customfuzz3: Running 1 inputs 1 time(s) each.
Running: /work/pov_input
malloc(): corrupted top size
==20== ERROR: libFuzzer: deadly signal